common/sshtools.py

```diff
@@ -42,7 +42,7 @@ class SSH(mount.MountControl):
 
     CHECK_FUSE_GROUP = True
 
-    def __init__(self, cfg = None, profile_id = None, hash_id = None, tmp_mount = False, parent = None, symlink = True, **kwargs):
+    def __init__(self, cfg = None, profile_id = None, hash_id = None, tmp_mount = False, parent = None, symlink = True, read_only = True, **kwargs):
```

Comment:
Sorry for the mess without documentation. I promise to get better. Already working on it :v: